.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  schedule:
+    # Use <https://crontab.guru> to conveniently edit cron schedule.
+    - cron: "0 7 * * *" # At 07:00 UTC every day.
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    env:
+      RUST_BACKTRACE: 1
+    strategy:
+      matrix:
+        build: [linux64, macos, win32]
```

Comment:
Couldn't you use `os: [ubuntu-latest, macos-latest, windows-latest]` here and do away with the `build` variable entirely?

Reply:
You mean replacing `build` with `os`?
Could be if you want combination build jobs: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations

Reply:
Not replacing it obviously, also changing the value.

Reply:
Could be, but those value in `build` also are the names when displayed in github CI checks.
We might prefer win32 over windows-latest because in the future, 
we might add win64 or linux32 build.

Reply:
I see... Is this the usual way to structure these matrices?

Reply:
I often see alex projects doing like that: 
https://github.com/alexcrichton/cc-rs/blob/master/.github/workflows/main.yml
https://github.com/alexcrichton/tar-rs/blob/master/.github/workflows/main.yml
https://github.com/alexcrichton/xz2-rs/blob/master/.github/workflows/main.yml

Reply:
I see, makes sense. Thanks!